charts/cosmic-config-server/templates/deployment.yaml

```diff
@@ -34,16 +39,16 @@ spec:
           }]
     spec:
       containers:
-      - name: cosmic-config-server
-        image: {{ .Values.registry }}missioncriticalcloud/cosmic-config-server:{{ .Values.cosmic_config_server.image_tag }}
-        imagePullPolicy: IfNotPresent
+      - name: {{ .Chart.Name }}
+        image: {{ .Values.global.registry }}{{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
```

Comment:
We should review our _case_ policy, at the moment we have camel-cased and underscore separated.